app/components/DuringIcoCountDown.tsx

```diff
@@ -0,0 +1,120 @@
+import * as cn from "classnames";
+import * as React from "react";
+import { Button, Col, Row } from "react-bootstrap";
+import * as styles from "./DuringIcoCountDown.scss";
+import { BlueHexagon, IHexagonsStackProps, WhiteHexagon } from "./HexagonsStack";
+
+interface IMoneyProps {
+  raised: number;
+  neuMarkAmount: number;
+  neuMarkToEtherRatio: number;
+  investorsAccountCreated: number;
+}
+
+export const HexagonText = (props: IMoneyProps) => {
+  const { raised, neuMarkAmount, neuMarkToEtherRatio, investorsAccountCreated } = props;
+  return (
+    <div className={styles.text}>
+      <p className={styles.title}>Total funds commited</p>
+      <p className={styles.content}>
+        <strong className={styles.extraSize}>{raised}</strong> ETH
+      </p>
+
+      <p className={styles.title}>Investors accounts created</p>
+      <p className={styles.content}>
+        <strong>
+          {investorsAccountCreated}
+        </strong>
+      </p>
+
+      <p className={styles.title}>Neumarks generated</p>
+      <p className={styles.content}>
+        <strong>{neuMarkAmount}</strong> <span>NEU</span>
+      </p>
+
+      <p className={styles.title}>Reward</p>
+      <p className={styles.content}>
+        <strong>{neuMarkToEtherRatio}</strong> <span> NEU / </span> <strong>1</strong>{" "}
+        <span>ETH</span>
+      </p>
+    </div>
+  );
+};
+
+export const HexagonsStack: React.SFC<IHexagonsStackProps> = ({
```

Comment:
Please extract this component to it's own file (create separate file for styles as well). Name it idk, foe example: `BlueWhiteHaxaghonStack`

Reply:
Or even better merge it with already existing `HeaxgonsStack` component and add prop like "mode" or "color" or just "invert" to select one of them